packages/@guidesmiths/cuckoojs-schematics/src/helm/files/base/.helm/values.yaml

```diff
@@ -0,0 +1,138 @@
+# Default values for <%= serviceName %>.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: <%= imageName %>
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+# Environment variables to be sended to our container
```

Comment:
Do we keep this comment as a help for the user to add env variables?

Reply:
Yes, these comments are to explain the values to be added to the helm chart